src/main/java/worker/Frontend.java

```diff
@@ -19,8 +20,12 @@
 
 
   ActorRef masterProxy = getContext().actorOf(
-      ClusterSingletonProxy.defaultProps("/user/master/active", "backend")
-      , "masterProxy");
+      ClusterSingletonProxy.props(
+          "/user/master",
+          ClusterSingletonProxySettings.create(getContext().system())
+              .withSingletonName("active")
```

Comment:
Weird, not sure why I changed that, possibly because that is how it was done in the corresponding Scala template.